hibernate-core/src/test/java/org/hibernate/orm/test/boot/database/qualfiedTableNaming/DefaultCatalogAndSchemaTest.java

```diff
@@ -512,6 +514,15 @@ private void verifyDDLQualifiers(String sql) {
 		verifyOnlyQualifier( sql, SqlType.DDL, EntityWithExplicitQualifiersWithIncrementGenerator.NAME, expectedExplicitQualifier() );
 		verifyOnlyQualifier( sql, SqlType.DDL, EntityWithDefaultQualifiersWithEnhancedSequenceGenerator.NAME, expectedDefaultQualifier() );
 		verifyOnlyQualifier( sql, SqlType.DDL, EntityWithExplicitQualifiersWithEnhancedSequenceGenerator.NAME, expectedExplicitQualifier() );
+
+		if ( dbSupportsCatalogs && expectedDefaultCatalog != null ) {
+			verifyOnlyQualifier( sql, SqlType.DDL, "catalogPrefixedAuxObject",
+					expectedQualifier( expectedDefaultCatalog, null ) );
+		}
+		if ( dbSupportsSchemas && expectedDefaultSchema != null ) {
+			verifyOnlyQualifier( sql, SqlType.DDL, "schemaPrefixedAuxObject",
+					expectedQualifier( null, expectedDefaultSchema ) );
+		}
```

Comment:
it would be great to add this JIRA ticket to the `@TestForIssue` annotation as well